core/templates/dev/head/exploration_editor/preview_tab/preview_tab.html

```diff
@@ -14,3 +17,22 @@
     </table>
   </div>
 </div>
+
+<style>
+  .oppia-preview-state-warning {
+    position: absolute;
```

Comment:
nit: alphabetize these -- so, background-color, then border, then border-radius, etc. Makes it easier to find the value for a particular property.